test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -3008,6 +3009,32 @@ public async Task GetStatus_ShowInputFalse(string storageProvider)
             }
         }
 
+        [Fact]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        public async Task GetStatus_testeventTypes()

```

Comment:
Can you update the test name `GetStatus_TestEventTypes` to match the existing naming convention?